packages/flutter/lib/src/rendering/sliver_grid.dart

```diff
@@ -290,14 +290,16 @@ class SliverGridDelegateWithFixedCrossAxisCount extends SliverGridDelegate {
   /// Creates a delegate that makes grid layouts with a fixed number of tiles in
   /// the cross axis.
   ///
-  /// All of the arguments must not be null. The `mainAxisSpacing` and
-  /// `crossAxisSpacing` arguments must not be negative. The `crossAxisCount`
-  /// and `childAspectRatio` arguments must be greater than zero.
+  /// All of the arguments except [mainAxisExtent] must not be null.
```

Comment:
This is looking good. Let's add some more documentation here that explains the two different behaviors, using childAspectRatio (the default) and using mainAxisExtent.

Adding a dartpad example of each would be awesome and easy for folks to play with and see the difference in action, There are examples like that in the code that are wrapped in a `{@tool dartpad ...`

Reply:
On it :+1: